From: Harshula Jayasuriya <harshula@debian.org>
Date: Sat, 9 Jan 2016 14:46:58 +1100
Subject: m17n-config-remove-libtool-user-switches

Remove Libtool user switches from m17n-config as we don't ship *.la files.
---
 m17n-config.in | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/m17n-config.in b/m17n-config.in
index 9977e5c..8bdfa92 100644
--- a/m17n-config.in
+++ b/m17n-config.in
@@ -10,7 +10,7 @@ exec_prefix=@exec_prefix@
 
 help ()
 {
-  echo "Usage: $0 [API-LEVEL ...] [--version | --cflags | --libs | --libtool]"
+  echo "Usage: $0 [API-LEVEL ...] [--version | --cflags | --libs]"
   echo "  API-LEVEL is CORE, SHELL (default), GUI, or FLT"
 }
 
@@ -79,12 +79,6 @@ case $1 in
   if test "@includedir@" != "/usr/include"; then
     echo "-I@includedir@"
   fi;;
-
---libtool)
-  for name in $LIBTOOLNAME; do
-    echo -n "@libdir@/lib${name}.la "
-  done;
-  echo;;
 *)
     help
     exit 1;;
@@ -99,7 +93,7 @@ cat > /dev/null <<EOF
 /***
 @section m17n-config-synopsis SYNOPSIS
 
-m17n-config [API-LEVEL ...] [--cflags | --libs | --libtool] [--version]
+m17n-config [API-LEVEL ...] [--cflags | --libs] [--version]
 
 @section m17n-config-description DESCRIPTION
 
@@ -116,8 +110,6 @@ The other arguments are as follows.
 	Print compile option (e.g. -I/usr/local/include)
 - --libs\n
 	Print link option (e.g. -L/usr/local/lib -lm17n)
-- --libtool\n
-	Print libtool option (e.g. /usr/local/lib/libm17n.la)
 - --version\n
 	Print version number of the m17n library.
 */ 
